NEWS.rst

```diff
@@ -4,66 +4,233 @@
 User-Visible Changes in Tahoe-LAFS
 ==================================
 
-Release ?? (??)
-'''''''''''''''
-
-unedited list of changes since last release, needs cleanup, explanation,
-credit authors, limit to user-visible things
-
-* add "tahoe list-aliases --readonly-uri" PR400
-* web: return "Referrer-Policy: no-referrer" on all requests
-* deps: use stdlib "json" instead of external "simplejson" #2766 PR405
-* internal: use @implementer instead of implements PR406
-* add "tahoe status" command
-* other PRs: 407 412 410 419 423
-* deps: require txi2p>=0.3.2 to work around TLS who-is-client issue #2861 PR409
-* complain more loudly in setup.py under py3 PR414
-* immutable upload now uses happiness-metric algorithm #1382 PR416
-* deps: now need libyaml-dev from system before build PR420
-* rename "filesystem" to "file store" #2345 PR380
-* test "tahoe list-aliases --readonly-uri" #2863 PR403
-* replace deprecated twisted.web.client with treq #2857 PR428
-* internal: improve happiness integration test #2895 PR432
-* web internal: refactor response-format (?t=) logic #2893 PR429
-* web: add JSON welcome page #2476 PR430
-* improve/stablize some test coverage #2891
-* internal: fix pyflakes issues #2898 PR434
-* internal: setup.py use find_packages #2897 PR433
-* add magic-wormhole -based "tahoe invite" and "tahoe create-client --join="
-  #126 PR418
+Release 1.13.0 (18-May-2018)
+''''''''''''''''''''''''''''
+
+New Features
+------------
+
+The ``tahoe list-aliases`` command gained the ``--readonly-uri``
+option in `PR400`_, which lists read-only capabilities (the default
+shows read/write capabilities if available). This command also gained
+a ``--json`` option in `PR452`_, providing machine-readable output.
+
+A new command ``tahoe status`` is added, showing some statistics and
+currently active operations (similar to the ``/status`` page in the
+Web UI).
+
+Immutable uploads now use the "servers of happiness" algorithm for
+uploading shares. This means better placement of shares on available
+servers. See `PR416`_.
+
+To join a new client to a grid, the command ``tahoe invite`` was
+added. This uses `magic wormhole`_ to connect two computers and
+exchange the required information to start the client. The "client
+side" of this command is the also new option ``tahoe
+create-client --join=``. Together, this provides a way to provision a
+new client without having to securely transmit the fURL and other
+details. `PR418`_
```

Comment:
maybe "and automatically securely transmit ..." instead of "without having to securely transmit"?